4/jammy32/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:22.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN dpkg --add-architecture i386 \
+    && apt-get update \
+    && apt-get -y upgrade \
+    && apt-get install --no-install-recommends -y \
+        ca-certificates \
+        ccache \
+        clang \
+        cmake \
+        g++-multilib \
+        gcc-multilib \
+        git \
+        libopenal-dev:i386 \
+        libpng-dev:i386 \
+        libsdl2-dev:i386 \
+        libsdl2-mixer-dev:i386 \
```

Comment:
we dont need mixer anymore. Its fully been replaced by openal. Also any chance gtest could be added?

Reply:
sure

Reply:
done